src/dropbearkey.c

```diff
@@ -330,7 +335,7 @@ static int printpubfile(const char* filename) {
 	return err;
 }
 
-static void printpubkey(sign_key * key, int keytype) {
+static void printpubkey(const char * filename, sign_key * key, int keytype, const char * comment) {
```

Comment:
Maybe change `filename` to  `save_filename` or something to make it clearer that it's an output? Moving it to the final argument also might be better, inputs then outputs as a very rough rule.